### ISSUE TITLE
Generate an x86 MCode using a 64-bit GCC

### ISSUE THREAD

**Issue by Aldrin-John-Olaer-Manalansan** [joedf/MCode4GCC#4]
@joedf  I am trying to generate an MCode for both x86 and x64 but it seems this compiler doesn't have an option to choose what Assembly should be generated.

Suggestion:
- Add an option that allows the user to select between 32-bit or 64-bit MCode.
- if the user selects 32-bit MCode while using a 64-bit OS, the gcc **flags** will have an additional **-m32** to force gcc to compile the code into a 32-bit assembly.
- else no additional steps will be added. Since **-m32** flag only works on 64-bit gcc.

**Comment by joedf** [comment]
Yes, correct. For the moment, you can edit this line to add the flag: https://github.com/joedf/MCode4GCC/blob/de0ea3addc7477e82189ed04e2ea1dad47e31acc/MCode4GCC.ahk#L172C1-L172C57
Or you can maybe add the flag in the gcc/compiler path.

**Comment by Aldrin-John-Olaer-Manalansan** [comment]
> Yes, correct. For the moment, you can edit this line to add the flag: https://github.com/joedf/MCode4GCC/blob/de0ea3addc7477e82189ed04e2ea1dad47e31acc/MCode4GCC.ahk#L172C1-L172C57 Or you can maybe add the flag in the gcc/compiler path.

That's exactly what I did so a workaround isn't a problem for me but it's for your awareness for a formal implementation of such feature.

I find this tool a remarkable piece of idea and I think most people out there uses it. It would be best if some limits gets crossed out little by little.

**Comment by joedf** [comment]
Thanks. Yes, I shall make a quick update hopefully this weekend.
